app/views/users/sessions.py

```diff
@@ -141,6 +146,77 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    main = []
+    data = ["Speaker Name", "Speaker Session(s)", "Speaker Email", "Speaker Status", \
+            "Speaker Mobile", "Speaker Organisation", "Speaker Position"]
+    main.append(data)
+    for session in sessions:
+        for speaker in session.speakers:
+            data = []
+            data.append(speaker.name) if speaker.name else data.append('')
+            data.append(session.title) if session.title else data.append('')
+            data.append(speaker.email) if speaker.email else data.append('')
+            if speaker.sessions:
+                for sess in speaker.sessions:
+                    data.append(sess.state)
+            data.append(speaker.mobile) if speaker.mobile else data.append('')
+            data.append(speaker.organisation) if speaker.organisation else data.append('')
+            data.append(speaker.position) if speaker.position else data.append('')
+        main.append(data)
```

Comment:
And don't you think this should be _inside_ the loop ? Otherwise you're just inserting the last record. 

Please _check_ your code properly _before_ making a PR